tests/common/authpolicy/suite_test.go

```diff
@@ -127,7 +127,6 @@ func TestMain(m *testing.M) {
 	logger := log.NewLogger(
 		log.SetLevel(log.DebugLevel),
 		log.SetMode(log.ModeDev),
-		log.WriteTo(GinkgoWriter),
```

Comment:
Confused here. Writting to GinkgoWriter should be helpful as it only shows errors when the test fails and not when the test passes. That is desirable, isn't it??

Or maybe you want to have logs disabled entirely and only activate logs when I run one test?

Reply:
I think this is an issue with ginkgo personally, or maybe i just don't know how to use it.

You are correct, anything written to GinkgoWriter shouldn't appear unless it fails, that is unless you use the `-v` option on ginkgo and then it will spit everything out regardless.

Problem is the output without `-v` is rubbish as it doesn't even tell you what tests it's running, just dots. What i would like is the output you get with `-v`, without the logs unless you get an error but i can't figure out any magic combination to make that happen.

This is the lowest priority thing in this PR btw, I'd happily drop that for now. The other changes are more important.

Reply:
> What i would like is the output you get with -v, without the logs unless you get an error

+1

Reply:
> Problem is the output without -v is rubbish as it doesn't even tell you what tests it's running

I do not see it as a problem. What is the problem there? We can make it optional, something that you can enable/disable. 

Anyway. I am not blocking this in any way. Feel free to remove the configuration to write to  GinkgoWriter.

LGTM

Reply:
Being able to optionally `log.WriteTo(GinkgoWriter)` (default to `false`) would be nice, I guess. The otherwise noisy output sometimes helps us spot silent issues in advance, even when the tests succeed.

What about adding an env var to control it? (Willing to give up completely on the idea if you say it's too ugly of a solution.) We'd have to add it to the `workflow_dispatch` as a parameter too.

Reply:
Yeah, this outputting happened since moving to the policy-machinery (previous to this, it worked as it should by outputting logs only when a test fails).

So this looks good to me until we resolve why the policy machinery is causing this

Reply:
I have removed disabling the logs entirely from this PR, Will look at it separately, as i said it was the least important thing here.

Reply:
![image](https://github.com/user-attachments/assets/77fd6c95-d809-4fc6-ba71-b2312bbed546)

I guess it's not the worst, i just like the more verbose output where it shows you the tests cases :shrug: 

Should still get the logs on failure this way i guess.